components/application-configuration/ui/src/main/resources/PhenoTips/DBConfigurationClass.xml

```diff
@@ -193,7 +193,7 @@ $!value
       <unmodifiable>0</unmodifiable>
       <validationMessage/>
       <validationRegExp/>
-      <values>metric|imperial</values>
+      <values>metric=$services.localization.render('phenotips.DBConfigurationClass.metric')|imperial=$services.localization.render('phenotips.DBConfigurationClass.imperial')</values>
```

Comment:
Nope, just translate:

    PhenoTips.DBConfigurationClass_measurementUnitSystem_metric=
    PhenoTips.DBConfigurationClass_measurementUnitSystem_imperial=

You can then use these keys above, in the `customDisplay` section.